lib/Agrammon/Web/Service.pm6

```diff
@@ -271,14 +272,23 @@ class Agrammon::Web::Service {
         $model-path.IO.&child-secure($technical).slurp
     }
 
-    method get-outputs-from-csv(
-        Agrammon::DB::User $user,
-        Str $simulation-name, Str $dataset-name, $csv-data,
+    subset InputFormats of Str where  { $_ eq 'application/json' or $_ eq 'text/csv' };
+    subset OutputFormats of Str where { $_ eq 'application/json' or $_ eq 'text/csv' or $_ eq 'text/plain' };
```

Comment:
Missed this before, but: since in non-block form a `subset`'s `where` part is smart-matched against the value, you can write these was:

```
    subset InputFormats of Str where 'application/json' | 'text/csv';
    subset OutputFormats of Str where 'application/json' | 'text/csv' | 'text/plain';
```

Reply:
Nice. Done.